src/common/fiscal-year.test.ts

```diff
@@ -1,25 +1,136 @@
 import { DateTime } from 'luxon';
-import { fiscalYear } from '.';
-import { fullFiscalQuarter } from './fiscal-year';
+import {
+  endOfFiscalYear,
+  expandToFullFiscalYears,
+  fiscalQuarter,
+  fiscalYear,
+  fiscalYears,
+  fullFiscalQuarter,
+  fullFiscalYear,
+  startOfFiscalYear,
+} from './fiscal-year';
+import { DateInterval } from './temporal';
 
-describe('fiscal-year', () => {
+const Feb2019 = DateTime.fromObject({ year: 2019, month: 2, day: 1 });
+const Oct2019 = DateTime.fromObject({ year: 2019, month: 10, day: 1 });
+const Nov2019 = DateTime.fromObject({ year: 2019, month: 11, day: 1 });
+const Dec2019 = DateTime.fromObject({ year: 2019, month: 12, day: 1 });
+const Jan2020 = DateTime.fromObject({ year: 2020, month: 1, day: 1 });
+const Jun2020 = DateTime.fromObject({ year: 2020, month: 6, day: 1 });
+const Sep2020 = DateTime.fromObject({ year: 2020, month: 9, day: 1 });
+const Oct2020 = DateTime.fromObject({ year: 2020, month: 10, day: 1 });
+const Sep302019 = DateTime.fromObject({ year: 2019, month: 9, day: 30 });
+const Sep302020 = DateTime.fromObject({ year: 2020, month: 9, day: 30 });
+const Sep302021 = DateTime.fromObject({ year: 2021, month: 9, day: 30 });
+const dInterval2020 = DateInterval.fromObject({
```

Comment:
Our coding standard here for variable names is camelCase.  So for these, for example, `Feb2019` should become `feb2019`.

Reply:
updated variable names to camelCase